symbols/natural/volcano.svg

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
The [icon guildelines](https://github.com/gravitystorm/openstreetmap-carto/blob/master/CONTRIBUTING.md#map-icon-guidelines) say
> All new icons must be SVG format only. The SVG must be saved as standards compliant SVG without any proprietary tags. In Inkscape software, you will need to "Save As..." and choose the format Optimized SVG (preferable) or Plain SVG.

It doesn't look like that was done hyere by the amount of sodipodi stuff in the SVG.

Reply:
@ZeLonewolf If you want I have a simplified and pixel-aligned version in https://github.com/pitdicker/openstreetmap-carto/commit/3b89a05d7f3b9ad74739d158b0c4905d72fc04fb.

Reply:
Thanks @pitdicker!